client/src/components/Notification/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const Notification = (props) => {
+  function playAudio() {
+    const audio = document.getElementsByClassName('audio-component');
+    audio.play();
+  }
+  const [play, setPlay] = React.useState(props.play);
+
+  if (play) {
```

Comment:
This part of the code will never get executed once `play` is false. As this should react as prop changes, use useEffect hook.
State is not required.

```
useEffect(() => { 
    play && playAudio();
}, [play, audio]);
```